x-pack/plugins/enterprise_search/common/ui_settings_keys.ts

```diff
@@ -7,3 +7,4 @@
 
 export const enterpriseSearchFeatureId = 'enterpriseSearch';
 export const enableIndexPipelinesTab = 'enterpriseSearch:enableIndexTransformsTab';
+export const enableBehavioralAnalyticsSection = 'enterpriseSearch:enableBehavioralAnalyticsSection';
```

Comment:
@joemcelroy You will also need to add this key:

[src/plugins/kibana_usage_collection/server/collectors/management/schema.ts](https://github.com/TattdCodeMonkey/kibana/blob/main/src/plugins/kibana_usage_collection/server/collectors/management/schema.ts#L545-L548)
[src/plugins/kibana_usage_collection/server/collectors/management/types.ts](https://github.com/TattdCodeMonkey/kibana/blob/main/src/plugins/kibana_usage_collection/server/collectors/management/types.ts#L148)